specification/attestation/resource-manager/Microsoft.Attestation/2018-09-01-preview/examples/Create_AttestationProvider.json

```diff
@@ -0,0 +1,28 @@
+{
+    "parameters": {
+        "resourceGroupName": "sample-resource-group",
+        "serviceName": "sampleservicename",
+        "subscriptionId": "00000000-0000-0000-0000-000000000000",
+        "api-version": "2018-09-01-preview"
+    },
+    "responses": {
+        "201": {
+            "body": {
+                "id": "subscriptions/6c96b33e-f5b8-40a6-9011-5cb1c58b0915/resourceGroups/testrg/providers/Microsoft.Attestation/attestationProviders/codes1",
+                "name": "codes1",
+                "type": "Microsoft.Attestation/attestationProviders",
+                "properties": {
+                    "status": "Ready"
+                }
+            }
+        },
+        "409": {
```

Comment:
please remove 409 and provide example for 200. The response status codes should match the ones specified in the operation in the spec.